### PR TITLE
Add tagging on release

### LINKED ISSUE #850: Add Language-Settings.ps1 script...
Needed to onboard to common EngSys processes.

### PR DESCRIPTION
**Pull request by mitchdenny**
This PR adds ```Language-Settings.ps1``` which is used by the release tagging logic to detect artifacts that are being published by path. I've validated that this works using ```AzureTemplate```.

Closes #850

### CODE REVIEW

File: eng/scripts/Language-Settings.ps1 [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        }
+
+        Write-Host "VersionCheck to CocoaPods for packageId $PackageId failed with statuscode $statusCode"
+        Write-Host $statusDescription
+        exit(1)
+    }
+    return $false
+}
+
+# Parse out package publishing information given a maven CocoaPod spec files
```
Comment (tjprescott) [inline eng/scripts/Language-Settings.ps1:62]: Scratch maven

File: eng/scripts/Language-Settings.ps1 [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+function Get-swift-PackageInfoFromPackageFile ($pkg, $workingDirectory)
+{
+    $podspec = Get-Content -Raw -Path $pkg | ConvertFrom-Json
+    $pkgId = $podspec.name
+    $pkgVersion = $podspec.version
+    $docsReadMeName = $pkgId -replace "^azure-" , ""
```
Comment (weshaggard) [inline eng/scripts/Language-Settings.ps1:68]: Is this needed here?

File: eng/scripts/Language-Settings.ps1 [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+# Returns the maven (really sonatype) publish status of a package id and version.
+function IsCocoaPodsPackageVersionPublished($PackageId, $PackageVersion)
+{
+    $url = ComputeCocoaPodsSpecUrl -PackageId $PackageId -PackageVersion $PackageVersion
+
+    Write-Host "Checking $url"
+
+    try
+    {
+        $uri = "https://oss.sonatype.org/content/repositories/releases/$groupId/$pkgId/$pkgVersion/$pkgId-$pkgVersion.pom"
```
Comment (weshaggard) [inline eng/scripts/Language-Settings.ps1:33]: This looks like android not ios.

File: eng/scripts/Language-Settings.ps1 (hunk 2 above)
Comment (mitchdenny) [reply eng/scripts/Language-Settings.ps1:68]: Are you talking about ```$docsReadMeName```?

File: eng/scripts/Language-Settings.ps1 (hunk 3 above)
Comment (mitchdenny) [reply eng/scripts/Language-Settings.ps1:33]: Whoops. Not sure what happened there.

Comment (mitchdenny) [reply eng/scripts/Language-Settings.ps1]: Oh OK, I define the URl earlier and didn't remove this (and made a request against the wrong URL ... thanks for picking it up. I'll end-to-end test again).

**Comment by tjprescott** [comment]
@mitchdenny am I correct that this is only creating the tag for the mono-repo, not the SwiftPM mirror?

**Comment by mitchdenny** [comment]
That is correct, this is our standard tagging mechanism that we do in all pipelines. For tagging the mirror with ```x.y.z``` versions, it probably makes sense to do it in the sync repos script.

File: eng/scripts/Language-Settings.ps1 [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+$packagePattern = "*.podspec.json"
+
+function ComputeCocoaPodsSpecUrl($PackageId, $PackageVersion)
+{
+    # The CocoaPods spec repo is a repository for JSONified versions of the PodSpec file. The path to the JSON
+    # file is derived from the name of the package where the repo URL relative URL is:
+    #
+    #   Spec/x/y/z/[Package]/[Version]/[Package].podspec.json
+    #
+    # The x/y/z values are the first four characters from the MD5 hash of the package name. For example 4/4/a
```
Comment (weshaggard) [inline eng/scripts/Language-Settings.ps1:13]:
```suggestion
    # The x/y/z values are the first three characters from the MD5 hash of the package name. For example 4/4/a
```

File: sdk/template/AzureTemplate/CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,5 @@
 # Release History
+## 1.1.0
```
Comment (weshaggard) [inline sdk/template/AzureTemplate/CHANGELOG.md:2]: This doesn't seem to be using the common changelog logic

**Review by weshaggard — APPROVED** [review]

File: sdk/template/AzureTemplate/CHANGELOG.md (hunk 5 above)
Comment (mitchdenny) [reply sdk/template/AzureTemplate/CHANGELOG.md]: Fixed this up as well in the latest PR.

**Pull request merged**